.pre-commit-config.yaml

```diff
@@ -101,6 +101,11 @@ repos:
           - types-aiobotocore
           - watchdog
           - xxhash
+  - repo: https://github.com/ansible-community/antsibull-changelog
+    rev: 0.29.0
+    hooks:
+    - id: antsibull-changelog-lint
+    - id: antsibull-changelog-lint-changelog-yaml
```

Comment:
If you lint `changelogs/changelog.yaml`, you also need to add that file. Otherwise linting it will fail.